README.md

```diff
@@ -328,6 +335,36 @@ For pure cabal builds i.e. when `BUILD=cabal-v2` and `RESOLVER` is not
 specified, `cabal` and `ghc` must be pre-installed on the system before
 building.
 
+## Remote repository
+
+Packcheck provides simple tools to work with a remote repository. If
+`GIT_SOURCE` is non-empty then packcheck tries to clone it into
+`.packckeck.remote`. All the work is now done inside `.packcheck.remote`.
+`.packcheck` is now created inside `.packcheck.remote`.
+
```

Comment:
It is sneaky to create a hidden .packcheck.remote in the current directory. Especially because the command can be run from anywhere. We could be more explicit and clone the repo visibly in the current directory and then run packcheck inside that.

Reply:
We can maybe add a `GIT_CLONE_TO`.